github/release_notes/renderer.py

```diff
@@ -216,20 +217,16 @@ def _header_suffix_reference(
             if should_generate_link:
                 reference_id_text = rls_note_obj.reference.identifier[0:12] # short commit hash
 
-        reference = '{reference_prefix}{ref_id}'.format(
-            reference_prefix=reference_prefix,
-            ref_id=reference_id_text,
-        )
-
+        reference = f'{reference_prefix}{reference_id_text}'
         if is_reference_auto_linked:
             return reference
 
         if not should_generate_link:
             # returns e.g. gardener/cc-utils#42 or g. gardener/cc-utils@commit-hash
-            return '{repo_path}{reference}'.format(
-                    repo_path=rls_note_obj.cn_source_repo.github_repo_path(),
-                    reference=reference
-                )
+            return (
+                f'{rls_note_obj.source_component_access.org_name()}/'
```

Comment:
this is a bit cumbersome. how about adding a convenience-method to the access-type? (e.g. repository_full_name, as github calls it). also, how about using urljoin? I think, the formatstrings in this case make things harder to read than if using simple str-concats tbh